config/model_configs/sphere_baroclinic_wave_rhoe_topography_dcmip_rs_deep.yml

```diff
@@ -0,0 +1,13 @@
+topography: "DCMIP200"
+dt_save_state_to_disk: "6days"
+rayleigh_sponge: true
+topo_smoothing: true
+initial_condition: "DryBaroclinicWave"
+t_end: "6days"
+use_reference_state: false
+dt: "200secs"
+job_id: "sphere_baroclinic_wave_rhoe_topography_dcmip_rs"
+deep_atmosphere: true
+diagnostics:
+  - short_name: [pfull, wa, va, rv]
+    period: 1days
```

Comment:
I added this test to ensure we are using the deep sphere code, but if it doesn't make sense physically, we should remove it and put something else in place.

Reply:
Noted - @Sbozzolo can we tag a CC release? I'm tracking `main` for current tests locally but we should wait for a ClimaCore release since there are major changes to metric reconstruction methods for GPU compatibility.

Reply:
Yes!

Reply:
Let's add it to this one instead `sphere_held_suarez_rhoe_hightop`. It has a higher top and will show more clearly any effect (or issues) of the deep atmosphere (unless it is intentional to test deep atmosphere with topography? @akshaysridhar). We should add a longrun as well, e.g. this one `longrun_hs_rhoe_dry_55km_nz63`.